.build/build-owasp.xml

```diff
@@ -16,30 +16,49 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<project basedir="." name="apache-cassandra-owasp-tasks">
-    <property name="dependency-check.version" value="8.3.1"/>
-    <property name="dependency-check.home" value="${build.dir}/dependency-check-ant-${dependency-check.version}"/>
+<project basedir="." name="apache-cassandra-owasp-tasks"
+         xmlns:unless="ant:unless">
+    <property name="dependency-check.version" value="9.0.2"/>
+    <property name="dependency-check.home" value="${tmp.dir}/dependency-check-ant-${dependency-check.version}"/>
```

Comment:
I am personally putting everything into `/home/user/.cache/cassandra/...`. If I was about to do that here, by modifying /home/user/.ant/build.properties, how that would look like? It would be great if `${tmp.dir}` here was actually some other property, like `${dependency-check.home.root}` or similar, because when I override `${dependency-check.home}` in `build.properties`, I do not know what `${dependency-check.version}` is ... it might change over time. Does this make sense?